File: backend/components/metiers/administration_publique/guides/core/fixtures/guides/guide_fixtures.test.js
// guide_fixtures.test.js – Tests unitaires JS pour guide_fixtures.js
const { getFixturesGuide } = require('./guide_fixtures');
describe('guide_fixtures.js', () => {
  it('doit retourner un guide de fixtures complet', () => {
    const guide = getFixturesGuide();
    expect(guide.title).toBe('Guide Fixtures 3D');
    expect(Array.isArray(guide.bestPractices)).toBe(true);
    expect(guide.bestPractices).toContain('Utiliser des fixtures typées et documentées');
    expect(Array.isArray(guide.integrationSteps)).toBe(true);
    expect(guide.integrationSteps).toContain('Importer les fixtures via le point d’entrée');
  });
});

File: backend/components/metiers/administration_publique/guides/core/fixtures/guides/guide_fixtures.test.py
"""
Tests ultra avancés pour guide_fixtures.py (guides/core/fixtures/guides)
"""
from .guide_fixtures import get_fixtures_guide

def test_get_fixtures_guide():
    guide = get_fixtures_guide()
    assert isinstance(guide, dict)
    assert guide['title'] == 'Guide Fixtures 3D'
    assert 'bestPractices' in guide
    assert 'integrationSteps' in guide
    assert 'Utiliser des fixtures typées et documentées' in guide['bestPractices']
